src/main/java/com/neuvector/Scanner.java

```diff
@@ -106,6 +113,7 @@ public static ScanRepoReportData scanLocalImage(Image image, NVScanner nvScanner
             errorMessage = "The image and nvScanner can't be null.";
         }
 
+        String dockerGroupCmdArgs[] = getDockerGroupCmdArgs(getScanReportPath(nvScanner.getNvMountPath()));
```

Comment:
Arrays in Java usually declared `String[] variableName`, not `String variableName[]`...